Makefile

```diff
@@ -279,7 +279,7 @@ uwsgi: collectstatic
 	@if [ "$(VENV_BASE)" ]; then \
 		. $(VENV_BASE)/awx/bin/activate; \
 	fi; \
-    uwsgi -b 32768 --socket 127.0.0.1:8050 --module=awx.wsgi:application --home=/venv/awx --chdir=/awx_devel/ --vacuum --processes=5 --harakiri=120 --master --no-orphans --py-autoreload 1 --max-requests=1000 --stats /tmp/stats.socket --lazy-apps --logformat "%(addr) %(method) %(uri) - %(proto) %(status)" --hook-accepting1-once="exec:awx-manage run_dispatcher --reload"
+    uwsgi -b 32768 --socket 127.0.0.1:8050 --module=awx.wsgi:application --home=/venv/awx --chdir=/awx_devel/ --vacuum --processes=5 --harakiri=120 --master --no-orphans --py-autoreload 1 --max-requests=1000 --stats /tmp/stats.socket --lazy-apps --logformat "%(addr) %(method) %(uri) - %(proto) %(status)"
```

Comment:
If I left this in, then tests that restarted services would start to get 502 errors half-way through. The expectation for normal production machines is that the dispatcher can be restarted without having this happen to web requests. If we could still enable this for honcho, but not for supervisor, that would probably be best, I agree.

Reply:
I'm kind of interested in pursuing that further

https://uwsgi-docs.readthedocs.io/en/latest/ParsingOrder.html

But I need to sync up with everyone about it first.

Reply:
I removed that so that it won't block the rest of this work. Will work on it separately

https://github.com/ansible/awx/compare/devel...AlanCoding:uwsgi_config?expand=1